app-policy/Dockerfile.amd64

```diff
@@ -1,67 +0,0 @@
-# Copyright (c) 2018-2019 Tigera, Inc. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-ARG GIT_VERSION=unknown
-ARG UBI_IMAGE
-
-FROM ${UBI_IMAGE} as ubi
-
-RUN mkdir -p /tmp/dikastes
-RUN chmod 0777 /tmp/dikastes
```

Comment:
`0777` is too open and doesn't look like it is needed. Revert back to the default `0755` when creating/copying the `/var/run/dikastes` folder in the unified `Dockerfile`.